src/index.js

```diff
@@ -0,0 +1,15 @@
+import {
+    clearCompleted,
+    editTodoList,
+    formSubmission,
+    populateTasks,
+  } from './modules/editTask.js';
+  import './index.css';
+  
+  const clearCompletedButton = document.querySelector('.clear-completed');
+  
+  clearCompletedButton.addEventListener('click', clearCompleted);
+  
+  formSubmission();
+  editTodoList();
+  populateTasks();
```

Comment:
Great job implementing the import function and calling all the functions inside a index.js file.